packages/editor/src/lib/editor/Editor.ts

```diff
@@ -8437,15 +8807,20 @@ export class Editor extends EventEmitter<TLEventMap> {
 			this._ctrlKeyTimeout = setTimeout(this._setCtrlKeyTimeout, 150)
 		}
 
-		const { originPagePoint, originScreenPoint, currentPagePoint, currentScreenPoint } = inputs
+		const { originPagePoint, currentPagePoint } = inputs
 
 		if (!inputs.isPointing) {
 			inputs.isDragging = false
 		}
 
+		const instanceState = this.store.unsafeGetWithoutCapture(TLINSTANCE_ID)!
+		const pageState = this.store.get(this._getCurrentPageStateId())!
+		const cameraOptions = this._cameraOptions.__unsafe__getWithoutCapture()!
+		const camera = this.store.unsafeGetWithoutCapture(this.getCameraId())!
```

Comment:
This doesn't matter too much, but since we know that we're not capturing dependencies in flush events, we can skip the checks for dependencies. It's a micro optimization.

Computed values normally capture their dependencies, which allows for the computed value to be recomputed when dereferenced after one of those dependencies has changed. The "unsafe get without capture" behavior allows for a value to be ignored.

For example:

```tsx
@computed function doubleShapeCount() {
  return this._shapeCount.get() * 2
}

const Example = track(() => {
  const editor = useEditor()
  return <div>{editor.doubleShapeCount.get()}</div>
}
```

The Example component would update whenever `_shapeCount` changed.

Whereas...

```tsx
@computed function doubleShapeCount() {
  return this._shapeCount.__unsafe__getWithoutCapture() * 2
}

const Example = track(() => {
  const editor = useEditor()
  return <div>{editor.doubleShapeCount.get()}</div>
}
```

The Example component would never update, even when `_shapeCount` changed, because that wasn't captured as a dependency.

However if we were only interested in the current value when we could skip a bit of logic by just asking for the current value.

```tsx
const Example = track(() => {
  const editor = useEditor()
  return <button onClick={() => console.log(editor._shapeCount.__unsafe__getWithoutCapture() * 2}/>
}
```

Reply:
Cool, gotcha - thanks for the explanation! Is the naming of `unsafe` is perhaps too strong a word? Like, it's definitely a deterrent to anybody reaching for that function but it looks like we're doing something hacky here when maybe it isn't. Should this be less of an `unsafe` but more something like `getWithoutCapture` but marked as `@internal`? Or it could be `getOnlyInstantaneousValueWithoutCapture` or something more wordy like that :P